dashboard/settings.py

```diff
@@ -35,6 +35,9 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'dashboard.socialaccount.providers.fxa',
+    'landing',
+
```

Comment:
any reason for this change? I tend to do my `INSTALLED_APPS` in this order:
#. django core
#. 3rd-party django apps/libraries
#. my apps/libraries